x-pack/plugin/transform/qa/multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/transform/integration/TransformIntegTestCase.java

```diff
@@ -140,6 +142,26 @@ protected StartTransformResponse startTransform(String id, RequestOptions option
         }
     }
 
+    // workaround for https://github.com/elastic/elasticsearch/issues/62204
+    protected StartTransformResponse startTransformWithRetryOnConflict(String id, RequestOptions options) throws Exception {
+        int retries = 10;
+        ElasticsearchStatusException lastConflict = null;
+        while (retries > 0) {
+            try (RestHighLevelClient restClient = new TestRestHighLevelClient()) {
+                return restClient.transform().startTransform(new StartTransformRequest(id), options);
+            } catch (ElasticsearchStatusException e) {
+                if (e.status().equals(RestStatus.CONFLICT) == false) {
```

Comment:
[nit] It's rather style issue, not a real problem here, but I would write this line:
```
                if (RestStatus.CONFLICT.equals(e.status()) == false) {
```

so that NPE is impossible (even theoretically).